packages/blade/src/utils/getMotionRefs.ts

```diff
@@ -0,0 +1,17 @@
+import type React from 'react';
+import type { MotionMetaProp } from '~components/BaseMotion';
+import type { BladeElementRef } from './types';
+
+type MotionRefsType = {
+  ref: React.Ref<BladeElementRef>;
+} & MotionMetaProp;
+
+const getOuterMotionRef = ({ _motionMeta, ref }: MotionRefsType): any => {
```

Comment:
let's add some details/docs and reasoning for these two functions, right now someone looking at this code have no idea what is `isEnhanced` why it's there, why we are using `innerRef` where did it come from etc etc. 

Let's properly document these